src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-upgrade-notes-lts-2021.md

```diff
@@ -126,6 +126,38 @@ Since [ELEMENTS-1124](https://jira.nuxeo.com/browse/ELEMENTS-1124) and [NXP-2869
 
 In LTS 2019, this change is already [available as an option](https://jira.nuxeo.com/browse/WEBUI-291). Feel free to activate it to test the editor and to allow your users to get accustomed to it before migrating.
 
+### Upgrade WebdriverIO v7
+
+The new version of Web UI introduces an upgrade on Webdriver IO, to its latest version, v7. With this upgrade there are a few things to consider when migrating:
```

Comment:
>to its latest version, v7

Looking a bit ahead and assuming that v8 will come out at some point, should we word it like that?

Reply:
I guess we can remove the version, yes.